datashare-app/src/main/java/org/icij/datashare/utils/PayloadFormatter.java

```diff
@@ -8,7 +8,16 @@
 public class PayloadFormatter {
```

Comment:
Maybe this class can be `static` so that we can use it without instantiate it in the other classes

Reply:
Agree, thanks. Changing now.